common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -34,6 +37,30 @@ def _is_finished_loading():
         EmptyPromise(_is_finished_loading, 'Finished rendering the advanced policy items.').fulfill()
         return self.q(css='body.advanced').present
 
+    @property
+    def key_names(self):
+        """
+            Returns a list of key names of all settings.
+        """
+        return self.q(css=KEY_CSS).text
+
+    @property
+    def deprecated_settings_button_text(self):
+        """
+            Returns text for deprecated settings button
+        """
+        return self.q(css=DEPRECATED_SETTINGS_BUTTON_SELECTOR).text[0]
+
+    def click_deprecated_settings_button(self):
```

Comment:
Nit: Would it be clearer if this were named `toggle_deprecated_settings_visibility`, so that it would be clearer from the method name what the result will be?

Reply:
Also, I notice that this method isn't used in the rest of this PR, so maybe it should just be moved to the same PR as the test-code that needs it.?